lms/static/coffee/src/instructor_dashboard/data_download.coffee

```diff
@@ -186,23 +186,76 @@ class ReportDownloads
       rowHeight: 30
       forceFitColumns: true
 
+
     columns = [
-      id: 'link'
-      field: 'link'
-      name: gettext('File Name')
-      toolTip: gettext("Links are generated on demand and expire within 5 minutes due to the sensitive nature of student information.")
-      sortable: false
-      minWidth: 150
-      cssClass: "file-download-link"
-      formatter: (row, cell, value, columnDef, dataContext) ->
-        '<a href="' + dataContext['url'] + '">' + dataContext['name'] + '</a>'
+      (
+        id: 'link'
+        field: 'link'
+        name: gettext('File Name')
+        toolTip: gettext("Links are generated on demand and expire within 5 minutes
+                          due to the sensitive nature of student information.")
+        sortable: false
+        minWidth: 150
+        cssClass: "file-download-link"
+        formatter: (row, cell, value, columnDef, dataContext) ->
+          '<a class="course-forums-data" href="' + dataContext['url'] + '">' + dataContext['name'] + '</a>'
+      ),
+      (
+        id: 'button'
+        field:'button'
+        name: ""
+        formatter: (row, cell, value, columnDef, dataContext) ->
+          if dataContext['name'].indexOf("course_forums") > -1
+            _.template('<div class="graph-forums"><i class="icon-pencil"></i> <%= label %></div>',
```

Comment:
wanna use this icon instead of the pencil? http://fortawesome.github.io/Font-Awesome/3.2.1/icon/bar-chart/